app/assets/stylesheets/core/_core.scss

```diff
@@ -1,205 +1,723 @@
-@import 'markus';
+// Core page styling
 
-@import 'navigation';
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 
+html,
+body {
+  min-height: 100%;
+  width: 100%;
+}
 
-#about_dialog {
-  width: 50%;
-  max-height: 75%;
+body {
+  background-color: $lighter-blue;
+  color: $dark-grey;
+  font: 400 0.8em/1.2em $fonts;
+  transition: background-color $time-quick;
 
-  #about_text {
-    max-height: 200px;
-    overflow: auto;
+  @include breakpoint(tiny) {
+    background-color: $white;
   }
 }
 
-#role_switch_dialog {
-  input[type='text'],
-  input[type='password'] {
-    margin-top: 1em;
-    padding: 0.5em 0.75em;
-    width: 100%;
+::selection {
+  background-color: $blue;
+}
+
+
+// Helper classes
+
+.hidden {
+  display: none;
+}
+
+.alignleft {
+  float: left;
+}
+
+.alignright {
+  float: right;
+}
+
+.clear-alignment {
+  clear: both;
+}
+
+
+// Main content
+
+.content {
+  background: $white;
+  border-bottom: 1px solid $blue;
+  padding: $dimen-vertical $dimen-horizontal;
+
+  @include breakpoint(mobile) {
+    margin-top: 3.5em;
+    padding: $dimen-vertical $dimen-horizontal-mobile;
   }
 }
 
-.image_input {
-  background: none;
-  border: none;
+.wrapper {
+  margin-top: 1em;
 }
 
-.ui-datepicker-current {
-  display: none !important;
+
+/** Code */
+pre,
+.code {
+  font: 14px/20px $mono-fonts;
```

Comment:
`14px/20px` should be written with a single space on each side of the operator: `14px / 20px`